assets/js/MenuItemLinks.js

```diff
@@ -0,0 +1,160 @@
+/*
+*   This content is licensed according to the W3C Software License at
+*   https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document
+*
+*   File:   MenuItemLinks.js
+*
+*   Desc:   Popup Menu Menuitem widget that implements ARIA Authoring Practices
+*/
+
+/*
+*   @constructor MenuItemLinks
+*
+*   @desc
+*       Wrapper object for a simple menu item in a popup menu
+*
+*   @param domNode
+*       The DOM element node that serves as the menu item container.
+*       The menuObj PopupMenu is responsible for checking that it has
+*       requisite metadata, e.g. role="menuitem".
+*
+*   @param menuObj
+*       The object that is a wrapper for the PopupMenu DOM element that
+*       contains the menu item DOM element. See PopupMenu.js
+*/
+var MenuItemLinks = function (domNode, menuObj) {
+
+  this.domNode = domNode;
+  this.menu = menuObj;
+
+  this.keyCode = Object.freeze({
```

Comment:
I am not sure how compatible is it with old browsers, maybe we should use simple objects.

Reply:
@andresmoschini this is part of the component files that W3C provides for the component to work.

Reply:
Si, lo sé, pero aún así no se que tan compatible es, y creo que ese cambio no hace a la funcionalidad, solo a las buenas prácticas.

Reply:
Com @damianmuti ya verificamos que _Object.freeze_ es muy compatible, no deberíamos preocuparnos.